### ISSUE TITLE
Pet bar doesn't disappear after temporary pets fade

### ISSUE THREAD

**Issue by RandomXD** [dalaranwow/dalaran-wow#5915]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
In some situation I noticed mostly during fights the pet bar from my pets still stays after the pets are gone, for example mirror images, treants from balance druid, i have screenshots for the druid/mage case but I know for sure it happens on enha sham with feral spirits, shadow priest with shadowfiend and dks with army/ghoul just I havent got a SS with those, maybe it can happen with other type of pets aswell, no idea.
**Expected behaviour**: Tell us what should happen instead
Pet bar should be gone the moment your pet/pets are gone.
**Steps to reproduce the problem**:
Pretty hard to reproduce, I tried to do it on dummies while either in combat or out of it and it was working normally but when im raiding it happens pretty often.
1. 
2. 
3. 

**Include proofs for this behaviour**
![WoWScrnShot_070920_221148](https://user-images.githubusercontent.com/36212275/87225822-56fba880-c398-11ea-8e63-ca6671475683.jpg)
thursday on my mage it happened again so I took a SS
![WoWScrnShot_071120_154704](https://user-images.githubusercontent.com/36212275/87225830-63800100-c398-11ea-905a-26254472fd9d.jpg)
today on my balance druid with treants too
You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
